newport/xps.go

```diff
@@ -492,6 +518,16 @@ func (xps *XPS) SetVelocity(axis string, vel float64) error {
 
 }
 
+// Stop aborts the motion of the axis
+func (xps *XPS) Stop(axis string) error {
+    cmd := fmt.Sprintf("GroupMoveAbort(%s)", axis)
+	resp, err := xps.openReadWriteClose(cmd)
```

Comment:
please run `gofmt` on the code; indentation should be consistent.  You can set up your editor to run gofmt every time you save the file for convenience